### PR TITLE
Hide unavailable session materials

### LINKED ISSUE #852: Nothing happens when you tap on the session materials
**Describe the bug**
Tap on session materials, there is a ripple but nothing happen

**To Reproduce**
1. Go to 'Session detail'
2. Scroll down to 'Material'
3. Click on 'Movie' / 'Slide'
4. Nothing happen

**Expected behavior**
Show the session materials only when they are available

**Screenshots**

https://user-images.githubusercontent.com/13198470/194111205-d657d616-64de-4624-b2e5-050686eaa45b.mp4

### PR DESCRIPTION
**Pull request by woxtu**
## Issue
- close #852 

## Overview (Required)
- The title says it all

## Links
- 

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
Can you remove the `Material` header as well when there is no material?

**Comment by woxtu** [comment]
@takahirom I'm done.

File: core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/TimetableAsset.kt [hunk 1]
```diff
@@ -6,4 +6,7 @@ import kotlinx.serialization.Serializable
 public data class TimetableAsset(
     val videoUrl: String?,
     val slideUrl: String?,
-)
+) {
+    val isAvailable: Boolean
+        get() = videoUrl != null || slideUrl != null
```
Comment (takahirom) [inline core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/TimetableAsset.kt:11]: 👍

**Review by takahirom — APPROVED** [review]

**Comment by takahirom** [comment]
LGTM!

**Pull request merged**